_internals/system-views.md

```diff
@@ -4,47 +4,47 @@ title: Babelfish system view
 nav_order: 4
 ---
 
-## Babelfish system views 
+## Babelfish system views
 
-Once in a while, an application has to access an MS SQL-compatible system view to
+Once in a while, an application has to access an MS SQL-compatible system view
+or a server side function to
 work properly. This is especially true for graphical user interfaces. However,
-accessing an MS SQL-compatible system view is also necessary for some applications 
+accessing an MS SQL-compatible system view is also necessary for some applications
 which have to collect information from the database to function properly.
 
 Providing support for system views is therefore important. The following views
```

Comment:
it is really not necessary to describe why catalog access may happen. The audience are database professionals, they understand this.

Reply:
A simpler (active voice) introduction to the section would probably read easier...  

You can query the following system catalog views to retrieve information about database objects and processes:

Reply:
You are right, of course.
Susan, I have taken your suggestion. Nice and succinct.